bin/ember-template-lint.js

```diff
@@ -232,6 +240,20 @@ function printPending(results, options) {
   }
 }
 
+function getTodoConfigFromOptions(options) {
+  let todoConfig = {};
+
+  if (Number.isInteger(options.todoDaysToWarn)) {
```

Comment:
When do we expect it _not_ to be a number? If it is just when it isn't passed, then I think you don't need the manual checks/gaurds (since yargs will "do the right thing").

So assuming `options` just has `undefined` for these props, couldn't you do:

```js
function getTodoConfigFromOptions(options) {
  return {
    warn: options.todoDaysToWarn,
    error: options.todoDaysToError,
  };
}
```

That _seems_ to give us the same thing? Or is this a case where you don't want the props on `options` at all if the options weren't specified?

Reply:
That's a good point, and we _could_ do that, but to your later point yes, I don't want to pass anything if they're not present. This means that we won't inadvertently set the values to undefined, which is what would happen.

Reply:
Seems good to me, dealers choice here 😸